### ISSUE TITLE
"Quick Menu" should be hidden in a text field when I type 

### ISSUE THREAD

**Issue by simonbcn** [ssborbis/ContextSearch-web-ext#710]
```
Google Chrome 
Context Search v1.46.26
```
Quick Menu enabled in text fields. The context menu should disappear if I type text or move the cursor (_with the mouse or with the keyboard_) but it does not always work, example:

https://github.com/ssborbis/ContextSearch-web-ext/assets/428231/feffba2c-8103-4d4d-b979-d9366b796eb8

**Comment by ssborbis** [comment]
I assume you're using the Auto method of launching the quick menu?

**Comment by ssborbis** [comment]
You're right that arrow keys do not close the menu. I'll check on that.

Can you confirm this option is unchecked in your settings?
![image](https://github.com/ssborbis/ContextSearch-web-ext/assets/24310997/9d52b07c-fb14-428c-a62d-7f2ea583e02f)

**Comment by simonbcn** [comment]
My options:
[ContextSearchOptions_2024-06-09_175914.json](https://github.com/user-attachments/files/15753599/ContextSearchOptions_2024-06-09_175914.json)

**Comment by ssborbis** [comment]
I think I found the issue. If you're in a position to do so, you could try the latest code in the master branch

**Comment by simonbcn** [comment]
It works if I type but not if I move the cursor with the arrow keys.

**Comment by ssborbis** [comment]
Arrow keys are currently used to navigate the menu. I can see about adding an option to disable them for that purpose.

**Comment by simonbcn** [comment]
But in case of input fields it does not work like that. In this case the arrow keys move the cursor through the text, not through the menu:
[Videocaptura de pantalla_20240708_205330.webm](https://github.com/ssborbis/ContextSearch-web-ext/assets/428231/0e69f37e-c2dc-4a50-8c88-8b8986928dfa)

**Comment by ssborbis** [comment]
You're right. I was thinking of another section of code. The issue here is event types. Arrow keys only register keydown / keyup events and I'm looking for input. This was somewhat intentional to not necessarily close the menu if non-character keys are being used. 

Anyway, I added a hidden setting `quickMenuCloseOnEditKeydown` you can set to `true` and close the menu with arrow keys, tab, etc
